src/python/foglamp/microservice_management/service_registry/instance.py

```diff
@@ -23,7 +23,7 @@ class Type(IntEnum):
 
         Storage = 1
         Core = 2
-        Device = 3
+        Southbound = 3
```

Comment:
looks fine; may be it should be *SouthBound*; but we can make search by type cAsE insensitive (sooner/ later point of time)!

Reply:
leave as Southbound see https://docs.google.com/a/dianomic.com/document/d/1MTFrwyXIa7YEPAIpnlNmDG7X1qFie9SL8PW2YT-hOkU/edit?disco=AAAABamR_ys

Reply:
Yeah, I saw that. Hence the comments around CamelCase thing and case insensitive search